src/components/Cards/WatchLaterCard/WatchLaterCard.jsx

```diff
@@ -0,0 +1,75 @@
+import { HiDotsVertical } from "react-icons/hi"
+import { BsFillEyeFill, BsCheckCircleFill } from "react-icons/bs"
+import { MdDelete } from "react-icons/md"
+import { useState } from "react"
+import {  useWatchLater } from "../../../contexts"
+import { shortString } from "../../../utils"
+
+
+export const WatchLaterCard = ({ video }) => {
+    const { thumbnail, title, description, creatorAvatar, creator
+        , views } = video;
+    const { removeFromWatchLater } = useWatchLater();
+
+
+    // overlay menu here
+    const [toolTip, setToolTip] = useState("showMenu");
+    const menuClickHandler = () => {
+        setToolTip((prevState) => prevState === "showMenu" ? setToolTip("hideMenu") : setToolTip("showMenu"))
+    }
+
+    return (
+        <div className="video__card">
+
+            {/* Card image here */}
+            <div className="video__card-img">
+                <img src={thumbnail} alt={`${title}`} className="image__res" />
+            </div>
+
+            {/*video data here  */}
+            <div className="video__card-data">
+                <div className="card__data-upper center__flex">
+                    <div className="data__upper-avatar center__flex">
+                        <img src={creatorAvatar} alt={`${creatorAvatar}`} className="image__res avatars" />
+                    </div>
+
+                    {/* Title here */}
+                    <div className="data-upper-title center__flex">
+                        <p>
+                            {shortString(title, 8)}
+                            {title.split(" ").length > 8 ? <span>...</span> : <span></span>}
+                        </p>
+                    </div>
+
+                    {/* Menu here */}
+                    <div className="card__data-menu" onClick={menuClickHandler} >
+                        <HiDotsVertical className="upper__title-menu" />
+                        <div className={`card__data-overlay ${toolTip}`}>
+                            {/* Menu icon here */}
+                            <MdDelete className="center__flex" onClick={() => removeFromWatchLater(video)} />
```

Comment:
title attribute can be added for icons for better understanding

Reply:
Okay, but I don't have any idea about title attribution...Can you explain to me a Lil bit so I can add them?

Reply:
it can be added like normal attribute. for ex- `  <MdDelete className="center__flex" title="delete" onClick={() => removeFromWatchLater(video)} />`.
This is useful in a case where only icons are used for an action. User can get to know the more info about the icon by hovering over it.
Note- if you want to add style to title. read about data-title